src/universal-express/index_spec.ts

```diff
@@ -0,0 +1,16 @@
+import { Tree } from '@angular-devkit/schematics';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+import * as path from 'path';
+
+
+const collectionPath = path.join(__dirname, '../collection.json');
+
+
+describe('sn', () => {
+  it('works', () => {
+    const runner = new SchematicTestRunner('schematics', collectionPath);
+    const tree = runner.runSchematic('my-schematic', {}, Tree.empty());
+
+    expect(tree.files).toEqual([]);
+  });
```

Comment:
Is it worth adding any other tests, like comparing the output to an example generated file?

Reply:
@jamesjwarren tests have been added: https://circleci.com/gh/thisissoon/schematics/2